gpkit/interactive/widget.py

```diff
@@ -30,17 +30,17 @@ def outputfn(gp):
             print gp.solution.table(["cost", "free_variables"])
 
     gp.sweep = {}
-    gp.prewidget = gp.last
+    # gp.prewidget = gp.last
 
-    solveargs["printing"] = False
+    solveargs["verbosity"] = 0
     def display(**subs):
-        gp.sub(subs, replace=True)
-        if hasattr(gp, "localsolve"):
-            gp.localsolve(**solveargs)
-        else:
-            gp.solve(**solveargs)
+        gp.substitutions.update(subs)
+        # if hasattr(gp, "localsolve"):
+        #     gp.localsolve(**solveargs)
+        # else:
+        gp.solve(**solveargs)
```

Comment:
Ah yes -- you are absolutely correct.

(This is a total hack/patch meant only to hold us over until #405 is resolved).